packages/components/src/toggle-group-control/toggle-group-control/component.tsx

```diff
@@ -83,12 +84,11 @@ function ToggleGroupControl(
 	const classes = useMemo(
 		() =>
 			cx(
-				styles.ToggleGroupControl,
+				styles.ToggleGroupControl( { size } ),
 				isBlock && styles.block,
-				'medium',
```

Comment:
This was adding a `.medium` class on the element. Not sure what that was for 🤔 There doesn't seem to be anything in our repo that uses it, at least. I'm inclined to remove it, especially when we're adding more size variants.

Reply:
It would be interesting to see if using `git-blame` we can understand if there was a specific reason for that hardcoded class to exist (just in case the removal causes any visual regression somewhere else in the codebase)

Reply:
There are no specific mentions of it in the [original PR](https://github.com/WordPress/gutenberg/pull/31937), but looking at the [component in G2](https://github.com/ItsJonQ/g2/blob/main/packages/components/src/SegmentedControl/SegmentedControl.js) we can surmise that the `medium` stuff is a [copy-paste artifact](https://github.com/ItsJonQ/g2/blob/60816e28df2ea7910997390ba16cd1d2d54c3197/packages/components/src/SegmentedControl/SegmentedControl.js#L52) from the [G2 sizing system](https://github.com/ItsJonQ/g2/blob/60816e28df2ea7910997390ba16cd1d2d54c3197/packages/components/src/SegmentedControl/SegmentedControl.styles.js#L123-L129).